src/infrastructure/implementations/manual-key-manager.ts

```diff
@@ -4,8 +4,10 @@ const log = getLogger(__filename);
 export class ManualKeyManager implements IKeyManager {
```

Comment:
No need to touch this class

Reply:
Because it implements IKeyManager `npm run checks` fails if function not implemented here

Reply:
Made it optional in interface and removed from here